.github/workflows/release.yaml

```diff
@@ -82,13 +83,19 @@ jobs:
                 # ArtifactHub annotation in Chart.yaml file
                 export CHANGELOG
                 yq -M -i '.annotations["artifacthub.io/changes"] = (load_str(strenv(CHANGELOG)) | capture("## Next release[\s]*(?P<changes>[\s\S]*?)[\s]*##") | .changes | split("\n") | .[] |= sub("^[\s]*-[\s]+", "") | filter(length > 0) | to_yaml)' $CHART_FILE
+                yq -M -i '(.dependencies[] | select(.name == "victoria-metrics-common") | .repository) = "https://victoriametrics.github.io/helm-charts"' $CHART_FILE
+                if [ -e $CHART_LOCK ]; then
+                  yq -M -i '(.dependencies[] | select(.name == "victoria-metrics-common") | .repository) = "https://victoriametrics.github.io/helm-charts"' $CHART_LOCK
+                fi
```

Comment:
It is seems like a `helm dep update` is needed after replacing repository URL. Otherwise, helm will throw a `digest` mismatch error.

Reply:
moved common chart dependency update to another PR, will create it as soon as changes from this PR merged and published